.github/ISSUE_TEMPLATE/7_custom.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Custom template
+about: I used to file Flutter issues and know exactly what I need to attach
+  to make an issue actionable.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<details>
+<summary></summary>
+
+</details>
```

Comment:
we don't need this, people can just use the "blank issue" link

Reply:
ok